app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,43 @@
  *= require_tree .
  *= require_self
  */
+
+ a:hover.link-defaut-color{
+  text-decoration: none !important;
+}
+
+.card-common {
+  box-shadow: 1px 2px 5px #999;
+  transition: all .4s;
+}
+
+.card-common:hover {
+  box-shadow:  2px 3px 15px #999;
+  transform: translateY(-1px);
+}
+
+.card {
+  border-radius: 5px;
+  -webkit-box-shadow: 0 1px 2.94px 0.06px rgba(4,26,55,0.16);
+  box-shadow: 0 1px 2.94px 0.06px rgba(4,26,55,0.16);
+  border: none;
+  margin-bottom: 30px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.card .card-block {
+  padding: 25px;
+}
+
+.order-card i {
+  font-size: 26px;
+}
+
+.f-left {
+  float: left;
```

Comment:
Float é difícil de dar manutenção. Ele tem comportamento imprevisível com mudanças de escopo maior.

Seria melhor usar `display: flex`: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
Removemos as classes f-left e f-float, experimentamos a utilização do display-flex, porém optamos por uma nova formatação com margin.

Reply:
Não sei se entendi. Vocês não conseguiram implementar o layout com a margin que gostariam usando display flex?

Reply:
Quando usamos o display: flex ficou um diferente porque o float: left era para forçar os campos à ficarem alinhados à esquerda. Porém, quando isso aconteceu e fomos tentar outras abordagens acabamos achando um layout melhor para o card.